examples/TestDriver/ios/TestDriver.xcodeproj/project.pbxproj

```diff
@@ -393,7 +393,7 @@
 			inputFileListPaths = (
 			);
 			inputPaths = (
-				"${SRCROOT}/Pods/Target Support Files/Pods-TestDriver/Pods-TestDriver-frameworks.sh",
+				"${PODS_ROOT}/Target Support Files/Pods-TestDriver/Pods-TestDriver-frameworks.sh",
```

Comment:
I'm guessing this is a byproduct of the different CP version - looks like they're probably equivalent when expanded.